src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/YesSqlOptions.cs

```diff
@@ -19,4 +19,6 @@ public class YesSqlOptions
     public IContentSerializer ContentSerializer { get; set; }
 
     public string TablePrefixSeparator { get; set; } = "_";
+
+    public bool UseLegacyIdentityColumn { get; set; } = true;
```

Comment:
I think that where @MikeAlhayek 's suggestion might apply. Store an enum, or the Type of the identity column. Today it is int32/int63 but maybe we'll support Guid someday.

Reply:
Exactly. We should not use bool in OC and should use the same Enum used in YesSql.

Reply:
No problem, but I'm not sure why it's named `IdentityColumnSize` not `IdentityColumnType`?